Samesite cookie workaround and https adopted by default

peppelinux opened the pull request:
README samesite cookie and https now are not a problem anymore

- feat: uwsgi as demo server!
- fix: allows_unsolicited come back to False by default
- example project have static file serve
- settings project now have security cookie parameters by default

peppelinux committed:
HTTPs by default

- feat: uwsgi as demo server!
- fix: allows_unsolicited come back to False by default
- example project have static file serve
- settings project now have security cookie parameters by default

peppelinux committed:
fix: README samesite cookie and https now are not a problem anymore!

peppelinux committed:
fix: CI gh actions

peppelinux committed:
fix: unit tests

peppelinux commented on src/djangosaml2_spid/tests.py:
```diff
@@ -166,37 +166,37 @@ def test_spid_public_cert(self):
+        # abs_path = finders.find('spid/logo.jpg')
+        # self.assertTrue(os.path.isfile(abs_path))
 
-    def test_spid_logo(self):
```
Hey @brunato 
I just omitted them, feel free to put them again (even in a next release).
Now also the demo server is a quite real one, with collected statics, tls certs and things.

peppelinux committed:
fix: gh actions with uwsgi ... experiment

peppelinux committed:
fix: gh actions pysaml2 dep

peppelinux committed:
fix: gh actions with uwsgi ... experiment 2

peppelinux committed:
feat: uwsg debug server with http keepalive

peppelinux committed:
fix: uwsgi in gh actions with silent stdout

brunato reviewed (changes requested):
Two things:
- I think _uwsgi_ can be added with a `pip install uwsgi` for the demo project but it shouldn't be added to app requirements. 
- Better remove commented test class if running these tests is problematic.

peppelinux commented:
uwsgi Is required because now it's the debug server for the example app.

It would removed when spid_django Will be a pure pypi app, until this ...

Ok on that unit test, I'll drop those comments

brunato commented:
This is already usable as reusable app even if it's not published on PyPI (see PR #33).
Adding _uwsgi_ webserver to `requirements.txt` force each user to install it in the SP's project also if uses another webserver.

brunato committed:
Update requirements files and readme

  - Update pysaml2 to peppelinux/pysaml2.git@pplnx-7.0.1
  - Include requirements.txt in requirements-dev.txt
